rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -1699,6 +1706,206 @@ impl PocketIcSubnets {
+            //     entries_buffer_limit = 10_000 : nat64;
+            //     module_hash = blob "\17\4d\7e\2c\3d\4d\3b\0f\34\61\63\4d\49\ea\c2\85\55\5a\97\48\de\7c\c7\a8\53\cf\ea\00\41\52\a1\97";
+            //     entries_fetch_limit = 1_000 : nat16;
+            //   };
+            //   canister_creation_cycles_cost = opt (0 : nat64);
+            //   analytics_config = opt opt variant {
+            //     Plausible = record {
+            //       domain = opt "identity.internetcomputer.org";
+            //       track_localhost = null;
+            //       hash_mode = null;
+            //       api_host = null;
+            //     }
+            //   };
+            //   related_origins = opt vec {
+            //     "https://id.ai";
+            //     "https://identity.ic0.app";
+            //     "https://identity.internetcomputer.org";
+            //     "https://identity.icp0.io";
+            //   };
+            //   feature_flag_continue_from_another_device = null;
+            //   captcha_config = opt record {
+            //     max_unsolved_captchas = 500 : nat64;
+            //     captcha_trigger = variant { Static = variant { CaptchaDisabled } };
+            //   };
+            //   dummy_auth = opt null;
+            //   register_rate_limit = opt record {
+            //     max_tokens = 25_000 : nat64;
+            //     time_per_token_ns = 1_000_000_000 : nat64;
+            //   };
+            // },
+            let internet_identity_test_args = InternetIdentityInit {
+                assigned_user_number_range: None, // DIFFERENT FROM ICP MAINNET
+                archive_config: None,             // DIFFERENT FROM ICP MAINNET
+                canister_creation_cycles_cost: Some(0),
+                register_rate_limit: Some(RateLimitConfig {
+                    max_tokens: 25_000,
+                    time_per_token_ns: 1_000_000_000,
+                }),
+                captcha_config: Some(CaptchaConfig {
+                    max_unsolved_captchas: 500,
+                    captcha_trigger: CaptchaTrigger::Static(StaticCaptchaTrigger::CaptchaDisabled),
+                }),
+                related_origins: None,  // DIFFERENT FROM ICP MAINNET
+                new_flow_origins: None, // DIFFERENT FROM ICP MAINNET
+                openid_google: Some(Some(OpenIdConfig {
+                    client_id:
+                        "775077467414-rgoesk3egruq26c61s6ta8bpjetjqvgo.apps.googleusercontent.com"
+                            .to_string(),
+                })),
+                analytics_config: None,     // DIFFERENT FROM ICP MAINNET
+                fetch_root_key: Some(true), // DIFFERENT FROM ICP MAINNET
+                enable_dapps_explorer: Some(false),
+                is_production: Some(false), // DIFFERENT FROM ICP MAINNET
+                dummy_auth: Some(None),
+                feature_flag_continue_from_another_device: None,
+            };
```

Comment:
Do these initial values make sense? In particular, disabling the archiving feature in local testing?

Reply:
> This is mainly for devs and their environment, right?

Indeed, it isn't supposed to be used in any production-like setting.